packages/mdx/mdx-hast-to-jsx.js

```diff
@@ -105,10 +105,10 @@ function toJSX(node, parentNode = {}, options = {}) {
       } class MDXContent extends React.Component {
   constructor(props) {
     super(props)
-    this.layout = ${layout}
+    this.layout = ${layout || 'null'}
```

Comment:
could also remove the entire constructor instead if layout isn't defined.

Reply:
I think this is semantically cleaner than omitting the constructor entirely, because it's communicating that there is no user layout. `null` is semantically different than `undefined`.